packages/fiori/src/WizardPopover.hbs

```diff
@@ -1,6 +1,9 @@
 <ui5-responsive-popover
 	horizontal-align="Center"
 	placement-type="Bottom"
+	role="dialog"
+	aria-modal="true"
```

Comment:
doesn't it have whose attributes already? what is more, it is not a good idea to set aria attributes on component level